OCMF-de.md

```diff
@@ -363,13 +355,13 @@ wünschenswert. Folgende sind dazu reserviert:
 
 #### Bezug von Seriennummern, Ladepunkt und Public-Key
 
-Zum Aufbau eines Bezugs zwischen dem zur Signaturprüfung zu benutzenden Public-Key und des
-Ladepunktes dienen eine oder mehrere Seriennummern, alternativ kann auch eine direkte Identifikation des
+Zum Aufbau eines Bezugs zwischen dem zur Signaturprüfung zu benutzenden Public-Key und dem
+Ladepunkt dienen eine oder mehrere Seriennummern, alternativ kann auch eine direkte Identifikation des
 Ladepunktes per ID erfolgen.
 
 Die Public-Keys zum Ladepunkt müssen auf anderem Wege als diesem Protokoll (Out-of-Band) an die Prüfkomponente in Verbindung mit der verwendeten ID oder den Seriennummern übermittelt werden, z.B. über ein zentrales Register.
 
-Um eine Signaturkomponente eindeutig zu identifizieren werden in diesem Protokoll folgende Merkmakle definiert:
+Um eine Signaturkomponente eindeutig zu identifizieren, werden in diesem Protokoll folgende Merkmakle definiert:
```

Comment:
While changing this line, please fix Merkmakle to Merkmale

Reply:
done